neuraxle/base.py

```diff
@@ -3316,6 +3325,42 @@ def __init__(self, cache_folder=None):
         ForceHandleMixin.__init__(self, cache_folder)
 
 
+class DefaultHandlerMethodMixin(ForceHandleOnlyMixin):
```

Comment:
Would this be right?
```suggestion
class IdentityHandlersMixin(ForceHandleOnlyMixin):
```